index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute page</title>
+</head>
+<body>
+    <section id="main">
```

Comment:
I recommend that you choose descriptive names for your sections; You can consider using name such as main-container rather than just main. Otherwise, good work